### ISSUE TITLE
Update UI

### ISSUE THREAD

**Issue by sudeepshukla930** [Nivedita-coder/webathon#1]
Hey @Nivedita-coder ,
While Scrolling through the github , I found this repo. 
I want to contribute something. 
can I update its UI??
Waiting for your response

**Comment by Nivedita-coder** [comment]
> Hey @Nivedita-coder ,
> While Scrolling through the github , I found this repo.
> I want to contribute something.
> can I update its UI??
> Waiting for your response

Yes, go ahead!

**Comment by Nivedita-coder** [comment]
> Hey @Nivedita-coder ,
> While Scrolling through the github , I found this repo.
> I want to contribute something.
> can I update its UI??
> Waiting for your response

**Comment by Nivedita-coder** [comment]
> Hey @Nivedita-coder ,
> While Scrolling through the github , I found this repo.
> I want to contribute something.
> can I update its UI??
> Waiting for your response

yes , go ahead but if u are doing this for hacktoberfest then this repo is not have  hacktoberfest label.

**Comment by sudeepshukla930** [comment]
Hey @Nivedita-coder , if you like the change , you can merge my pull request with **"hacktoberfest-accepted"** tag . It would be great honour for me.

**Comment by Nivedita-coder** [comment]
> Hey @Nivedita-coder , if you like the change , you can merge my pull request with **"hacktoberfest-accepted"** tag . It would be great honour for me.

okay sure!
